packages/2018-disaster-resilience/src/components/ViolentShakingAndGroundDeformation/index.js

```diff
@@ -1,25 +1,295 @@
+
+const dataURLs = [
+  'https://gist.githubusercontent.com/mendozaline/f78b076ce13a9fd484f6b8a004065a95/raw/ff8bd893ba1890a6f6c20265f720587f9595a9c4/pop.json',
+  'https://gist.githubusercontent.com/mendozaline/b3a75b40c9a60781b6adc77cebb9b400/raw/fa0aa13c75bfcc2fd92ccf1f3cc612af83d5d704/010-grocery.json',
+];
+
+class ViolentShakingAndGroundDeformation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      foundation1: null,
+      slide1: null,
+      error: null,
+    };
+  }
 
-export class ViolentShakingAndGroundDeformation extends React.Component {
   componentDidMount() {
-    // initialize data here
+    let cmp = this;
+    d3.queue()
+      .defer(d3.json, dataURLs[0])
+      .defer(d3.json, dataURLs[1])
+      .await((error, foundation1, slide1) => {
+        if (error) {
+          cmp.setState({ error: error });
+        }
+        cmp.setState({ foundation1, slide1 });
+      });
   }
 
   render() {
-    return (
-      <CivicStoryCard
-        title="Violent Shaking and Ground Deformation"
-        slug="violent-shaking-and-ground-deformation"
-      >
-        <Placeholder issue="217" />
-      </CivicStoryCard>
-    );
+    if (this.state.foundation1 === null) {
+      return null;
+    }
+    const { foundation1, slide1 } = this.state;
+    return dashboardComponent(foundation1, slide1);
   }
 }
 
-ViolentShakingAndGroundDeformation.displayName = 'ViolentShakingAndGroundDeformation';
+const dashboardComponent = (foundationData1, slideData1) => {
+  if (foundationData1 === null) {
+    return null;
+  }
+
+  const mapboxToken =
+    'pk.eyJ1IjoidGhlbWVuZG96YWxpbmUiLCJhIjoiY2o1aXdoem1vMWtpNDJ3bnpqaGF1bnlhNSJ9.sjTrNKLW9daDBIGvP3_W0w';
+  const mapboxStyle = 'mapbox://styles/hackoregon/cjiazbo185eib2srytwzleplg';
+  const planetColorScheme = [
+    [247, 244, 249, 255],
+    [231, 225, 239, 255],
+    [212, 185, 218, 255],
+    [201, 148, 199, 255],
+    [223, 101, 176, 255],
+    [231, 41, 138, 255],
+    [206, 18, 86, 255],
+    [152, 0, 67, 255],
+    [103, 0, 31, 255],
+  ];
+
+  const populationGetColor = f => {
+    const population = parseFloat(f.properties.total_population);
+    return population < 3000
+      ? planetColorScheme[0]
+      : population < 8000
+        ? planetColorScheme[1]
+        : population < 12000
+          ? planetColorScheme[2]
+          : population < 16000
+            ? planetColorScheme[3]
+            : population < 20000
+              ? planetColorScheme[4]
+              : population < 24000
+                ? planetColorScheme[5]
+                : population < 28000
+                  ? planetColorScheme[6]
+                  : population < 32000
+                    ? planetColorScheme[7]
+                    : planetColorScheme[8];
+  };
+
+  const foundation = {
```

Comment:
I mentioned this below, but I think some of these could be located in a constants file for reuse and set defaults there, and then as needed could overwrite values.

Reply:
Yeah, I think that a constants file is a good idea.

Reply:
Discussed and going to wait for the changes on the back-end